__tests__/components/editor/PropertyPanel.test.js

```diff
@@ -31,7 +31,6 @@ describe('<PropertyPanel />', () => {
   })
 
   it('generates a title', () => {
```

Comment:
if this is about popover title ... could it say so?  Seems like this test description is similarly confusing to me.

Reply:
PropertyPanel no longer generates a title or a popover. It renders a `PropertyTitle` and tests for only that. The test for  `PropertyTitle` tests for the specific kind.